packages/core-tester-cli/lib/commands/transfer.js

```diff
@@ -22,37 +22,36 @@ const sendTransactionsWithResults = async (transactions, wallets, transactionAmo
     return false
   }
 
+  if (!postResponse.data.transactionIds.length) {
+    return false
+  }
+
   for (const transaction of transactions) {
-    if (!postResponse.data.transactionIds.find(transactionId => (transaction.id === transactionId))) {
+    if (!postResponse.data.transactionIds.includes(transaction.id)) {
       logger.error(`Transaction '${transaction.id}' didn't get applied on the network`)
     }
   }
 
-  if (!postResponse.data.transactionIds.length) {
-    return false
-  }
-
   const delaySeconds = await utils.getTransactionDelay(transactions)
   logger.info(`Waiting ${delaySeconds} seconds for node to process and forge transfer transactions`)
   await delay(delaySeconds * 1000)
 
   const walletBalance = await utils.getWalletBalance(primaryAddress)
   logger.info('All transactions have been received and forged!')
 
-  if (walletBalance !== expectedSenderBalance) {
+  if (parseInt(walletBalance) !== parseInt(expectedSenderBalance)) {
     successfulTest = false
     logger.error(`Sender balance incorrect: '${walletBalance}' but should be '${expectedSenderBalance}'`)
   }
 
   wallets.forEach(async wallet => {
     const balance = await utils.getWalletBalance(wallet.address)
 
-    if (balance !== transactionAmount) {
+    if (parseInt(balance) !== parseInt(transactionAmount)) {
```

Comment:
I fixed this locally when i was working on a multisig issue on Friday and testing. I'd recommend changing getWalletBalance to return int here instead:

https://github.com/ArkEcosystem/core/blob/master/packages/core-tester-cli/lib/utils/get-wallet-balance.js#L7

also, you can do `return +wallet.balance` which casts as well, but whichever floats your boat.